### PR TITLE
fix: support air_temperature with ccs2

### PR DESCRIPTION
**Pull request by jwillemsen**
Currently not tested, just looked at the json data as attached
[kona2024data.txt](https://github.com/Hyundai-Kia-Connect/hyundai_kia_connect_api/files/14166280/kona2024data.txt)

### CODE REVIEW

**Review by cdnninja — CHANGES_REQUESTED** [review]
Unit should be set as well.

File: hyundai_kia_connect_api/KiaUvoApiEU.py [hunk 1]
```diff
@@ -406,7 +406,20 @@ def _update_vehicle_properties_ccs2(self, vehicle: Vehicle, state: dict) -> None
 
         vehicle.engine_is_running = get_child_value(state, "DrivingReady")
 
-        # TODO: vehicle.air_temperature = get_child_value(state, "Cabin.HVAC.Driver.Temperature.Value")
+        if get_child_value(state, "Cabin.HVAC.Row1.Driver.Temperature.Value"):
+            tempIndex = get_hex_temp_into_index(
```
Comment (cdnninja) [inline hyundai_kia_connect_api/KiaUvoApiEU.py:410]: To confirm is your raw data in hex? I got the impression it wasn't. This is only needed for cars that don't show the decimal value.

File: hyundai_kia_connect_api/KiaUvoApiEU.py [hunk 2]
```diff
@@ -406,7 +406,13 @@ def _update_vehicle_properties_ccs2(self, vehicle: Vehicle, state: dict) -> None
 
         vehicle.engine_is_running = get_child_value(state, "DrivingReady")
 
-        # TODO: vehicle.air_temperature = get_child_value(state, "Cabin.HVAC.Driver.Temperature.Value")
+        air_temp = get_child_value(
+            state,
+            "Cabin.HVAC.Row1.Driver.Temperature.Value",
```
Comment (cdnninja) [inline hyundai_kia_connect_api/KiaUvoApiEU.py:411]: Looks like no unit set now.

**Review by cdnninja — CHANGES_REQUESTED** [review]

File: hyundai_kia_connect_api/KiaUvoApiEU.py (hunk 2 above)
Comment (jwillemsen) [reply hyundai_kia_connect_api/KiaUvoApiEU.py:411]: This is just the retrieval into a local variable, it is set on line 415

File: hyundai_kia_connect_api/KiaUvoApiEU.py (hunk 1 above)
Comment (jwillemsen) [reply hyundai_kia_connect_api/KiaUvoApiEU.py]: You are correct, it is decimal, just found also that when the car is off the value is set to "OFF"

File: hyundai_kia_connect_api/KiaUvoApiEU.py (hunk 2 above)
Comment (cdnninja) [reply hyundai_kia_connect_api/KiaUvoApiEU.py:411]: Line 415 looks to be adjusting for off.

File: hyundai_kia_connect_api/KiaUvoApiEU.py (hunk 1 above)
Comment (cdnninja) [reply hyundai_kia_connect_api/KiaUvoApiEU.py]: If the value is off we will need to set to that a number as the data type isn't a string.

**Pull request merged**